INDEX

```diff
@@ -79,7 +103,7 @@
   },
   "node-red": {
     "MANIFEST": "node-red.json",
-    "description": "Low-code programming for event-driven applications.",
+    "description": "Node-RED is a programming tool for wiring together hardware devices, APIs and online services in new and interesting ways.",
     "icon": "https://icons.freenas.org/community-icons/node-red.png",
     "name": "Node-RED",
     "official": false
```

Comment:
We should have one for this plugin as well. Please let me know if you think i should perhaps do a clean up afterwards for all plugins ? Not sure how we have them in master :)